src/components/common/utils.css

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
See common above, should we use Basscss? Believe it will help with developer productivity since its OSS.